tests/code/test_fileHandler.py

```diff
@@ -35,6 +35,15 @@ def test_getFileFullPath():
     filePath = handledCourseFile.getFileFullPath(courseFileStorageObject[0].filename)
     assert filePath == 'app/static/files/courseattachments/1/coursefile.pdf'
 
+@pytest.mark.integration
+def test_makingdirectory():
+    #Testing that the file is created and it exists
+        
```

Comment:
We should make sure that the directory doesn't exist before we try making it, because otherwise we could be not creating it and we wouldn't know it.